telemetry.go

```diff
@@ -0,0 +1,31 @@
+package gocron
+
+import "github.com/prometheus/client_golang/prometheus"
+
+var (
+	jobLatency = prometheus.NewHistogramVec(prometheus.HistogramOpts{
+		Name: "gocron_job_latency_seconds",
+		Help: "The latency of jobs in seconds.",
+	}, []string{"job_name"})
+	// jobErrors = prometheus.NewCounter(prometheus.CounterOpts{
+	// 	Name: "gocron_jobs_failures_total",
+	// 	Help: "The total number of job failures.",
+	// })
+
+	metricsEnabled = false
```

Comment:
I think metrics should be tied/scoped to the scheduler. If someone has multiple schedulers running, might want to have metrics enabled on some but not others.